tests/Operation/ChangeStreamFunctionalTest.php

```diff
@@ -237,4 +237,22 @@ public function testConnectionException()
                         ]);
         $this->assertEquals($changeStreamResult->current(), $expectedResult);
     }
+
+    public function testMaxAwaitTimeMS()
+    {
+        $this->collection = new Collection($this->manager, $this->getDatabaseName(), $this->getCollectionName());
+        $maxAwaitTimeMS = 10;
+        $changeStreamResult = $this->collection->watch([], ['maxAwaitTimeMS' => $maxAwaitTimeMS]);
+        $changeStreamResult->next();
```

Comment:
I know you removed the `rewind()` call in the constructor, but can you confirm whether `$changeStreamResult->rewind()` invokes `getMore` and introduces a delay?

If so, we can do the assert and time check on that initial `rewind()`. It would also set a good example for anyone looking at the test files for guidance (ideally, users should rewind first).